src/docs/product/profiling/index.mdx

```diff
@@ -61,9 +61,24 @@ The "Slowest Functions" widget shows you a list of the functions that took the m
+
+Using the information on this page, you can dive deeper into what your code was executing during this transaction. The top area of the page includes a percentile chart with profile durations and a search bar. The chart is there so that you can visually identify possible performance regressions or changes - it supports selecting of ranges through a click-drag interaction. Use it to isolate certain date time ranges you would like to investigate. While selecting on date ranges is useful, it is important to be able to narrow down on the population as well, this is why the search bar filters are for. If for example you would like to narrow your investigation to profiles executed under a certain environment, apply the filter and the data will reflect that. Using both of these mechanisms allows you to narrow down on the time range and the population which might be exhibiting performance problems.
+
+Our bottom right side of the page has aggregated information about function regressions that Sentry may have detected during the selected time range as well as a list of the slowest functions in your application as well as some general information about the profile. We can see information like when was this profile last seen, p75, p95 and p99 values as well as the total amount of profiles we collected for this transaction.
+
+Below the general information, we have a list of regressions that Sentry may have detected as well as a list of the slowest functions executed during this transactions.
+Clicking on the before/after timings in the regressed function list will take you to example flamecharts of the before and after execution so that you can investigate what changed.
+
+The slowest functions list serves as an overview of functions which are consuming the most time in your application - these are usually good candidates for optimization.
+
+The left side of the page contains am aggregate flamegraph as well as digest of the slowest and most regressed functions. We recommend reading the docs about [Flame Graphs and Aggregated Flame Graphs](/product/profiling/flame-charts-graphs) so you can understand what this data is telling us. In short, looking for wide frames that have no children is a good first thing to look at - this will allow you to identify frames that are often occupying the stack and are good candidates for optimization.
+
+The alternative way you can also visualize the flamegraph is by clicking on the Call Tree button which will render the flamegraph as an inverted call tree (leafmost nodes first which expand to their parent nodes). The sample column in the call tree represent how often profiling collected stack samples where that function was on the stack. Expanding of the tree can be done via click interaction on the caret before the function name (holding ⌘ while doing so will expand that entire branch all the way to the root.)
```

Comment:
```suggestion
An alternative way to visualize the flamegraph is by clicking on the "Call Tree" button. This will render the flamegraph as an inverted call tree with its leftmost nodes first, which will then expand to its parent nodes. 

The sample column in the call tree represents the frequency with which profiling collected stack samples where a function was on a stack. Tree expansion can be done via click interaction on the caret before the function name (holding ⌘ while doing so will expand that entire branch all the way to the root.)
```

I might need you to walk me through this. Having a hard time finding it in the UI. This may also really benefit from an added visual.